.github/workflows/beaker.yml

```diff
@@ -100,7 +100,7 @@ jobs:
         if: ${{ inputs.rubocop }}
       - name: Setup Test Matrix
         id: get-outputs
-        run: bundle exec metadata2gha --domain ${{ inputs.domain }} --pidfile-workaround ${{ inputs.pidfile_workaround }} --beaker-facter "${{ inputs.beaker_facter }}"
+        run: bundle exec metadata2gha --domain ${{ inputs.domain }} --beaker-facter "${{ inputs.beaker_facter }}"
```

Comment:
:fire:

Reply:
Is that the reason for the change?  A lot of thing happen recently and I fail to follow… the PR description does not help :grin:

Reply:
Yeah. Podman doesn't need the pidfile workaround stuff, as it doesn't have the same bug as docker.